bookstore/clone.py

```diff
@@ -24,24 +22,24 @@ def initialize(self):
 
         self.session = aiobotocore.get_session()
 
-    async def _clone(self, s3_bucket, file_key):
-        path = file_key
+    async def _clone(self, s3_bucket, s3_key_file):
 
         self.log.info(f"bucket: {s3_bucket}")
-        self.log.info(f"key: {file_key}")
-        full_s3_path = s3_path(s3_bucket, path)
+        self.log.info(f"key: {s3_key_file}")
```

Comment:
I think this should probably be `s3_file_key` or `s3_object_key` rather than `s3_key_file`.

Reply:
It's odd that we force keys to be passed around rather than keeping credential collections purely within the s3 client. This makes setups where there's no key file unable to use this tool despite having valid access to S3.

Reply:
these aren't s3_keys. That's what I'm saying. They are paths to the object we should come up with a better name for them as they clearly just lead to confusion.

Reply:
Ah perhaps we should call it s3_object_key? that's the language they use in the s3 docs: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingMetadata.html#object-keys

Reply:
I could get behind `s3_object_key_name` which seems to accurately reflect the AWS docs and not imply that we are passing around keys or files.

Reply:
I went with `s3_object_key` as that is the way that it is referred to most generally in the docs and is a slight bit shorter. Should we change the url param to match? Since it hasn't been officially released yet, now is probably the time to do it.

Reply:
Yep. I would make it consistent. Thanks @mpacer

Reply:
I changed it in this PR.